exercises/concept/log-levels/.meta/config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "authors": [
+    {
+      "github_username": "mfoda",
+      "exercism_username": "mohammadfoda"
+    }
+  ],
+  "files": {
```

Comment:
I think you're missing the forked_from property. That property will be used to properly attribute reputation to the original author of the exercise.

Reply:
Ah, I missed that. Thanks for the reminder.

How do I know which concept this exercise is linked to in the fork? I can tell that this for `strings` since it's the only entry under `concepts` for this exercise in the csharp `config.json`. But that's an array and some exercises have multiple concepts. In that case what should the value be in `"forked_from": ["csharp/?"]` ?

Reply:
The other thing I just noticed is that the [spec](https://github.com/exercism/v3/blob/main/docs/concept-exercises.md#metaconfigjson) mentions that this field should be for the exercise that's forked from, but it seems that currently we're using the concept instead?

Reply:
> The other thing I just noticed is that the spec mentions that this field should be for the exercise that's forked from, but it seems that currently we're using the concept instead?

The `forked_from` should refer to `<track slug>/<exercise slug>`. We used to have the names of the exercises match the concepts, but we've since moved away from that. This is why there might be references to `csharp/strings`, even though that exercise has been renamed to `csharp/log-levels`. We just forgot to update the forked_from field (configlet will check its validity in the future).

Reply:
Aha, got it. Thanks for the clarification :-)